src/components/common/Extensions/UserNavigation/partials/WalletPopover/WalletPopover.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import { WalletPopoverProps } from './types';
+import PopoverBase from '~shared/Extensions/PopoverBase';
+import Icon from '~shared/Icon';
+import Link from '~shared/Link';
+import styles from './WalletPopover.module.css';
+import WalletPopoverOption from '../../WalletPopoverOption';
+import Button from '~shared/Extensions/Button';
+import { useAppContext, useMobile } from '~hooks';
+import TitledContent from '~common/Extensions/TitledContent/TitledContent';
+
+const displayName = 'common.Extensions.UserNavigation.partials.WalletPopover';
+
+const WalletPopover: FC<WalletPopoverProps> = ({ setTooltipRef, tooltipProps }) => {
+  const { formatMessage } = useIntl();
+  const isMobile = useMobile();
+  const { connectWallet } = useAppContext();
+
+  return (
+    <PopoverBase
+      setTooltipRef={setTooltipRef}
+      tooltipProps={tooltipProps}
+      classNames={styles.walletPopover}
+      withoutTooltipStyles
```

Comment:
I am not sure if it's a good name to tooltipStyles
As I can see in PopoverBase we have ` 'z-[9999] tooltip-container': !withoutTooltipStyles`

Can we change this name on `withTooltipStyles`and fix this conditional?

Reply:
Changed to `withTooltipStyles`